var/spack/repos/builtin/packages/rust/package.py

```diff
@@ -91,9 +94,33 @@ def setup_build_environment(self, env):
         ar = which("ar", required=True)
         env.set("AR", ar.path)
 
-        # Manually inject the path of openssl's certs for build.
-        certs = join_path(self.spec["openssl"].prefix, "etc/openssl/cert.pem")
-        env.set("CARGO_HTTP_CAINFO", certs)
+        # Manually inject the path of openssl's certs for build
+        # if certs are present on system via Spack or via external
+        # openssl.
+        def get_test_path(p):
```

Comment:
Why the name "get_test_path"?

Reply:
I think it's meant to be a shortened version of `get_and_test_path`, although maybe we should change it to make it more readable. (@trws is the original author. I just pulled this out of his larger developer tools PR.)

Reply:
Yup, that's right. I don't mind changing the name if you prefer something else.